provider/core/src/datagen/crabbake.rs

```diff
@@ -8,9 +8,23 @@ use crate::yoke::*;
 use alloc::boxed::Box;
 use crabbake::{Bakeable, CrateEnv, TokenStream};
 
+trait MyBakeable {
+    fn bake(&self, env: &CrateEnv) -> TokenStream;
```

Comment:
Suggestion: name the function something other than `bake` to reduce confusion with the real `Bakeable` trait